graph/docu/docu_04graph_04.cc

```diff
@@ -60,15 +70,14 @@ for (size_t e = 0; e < 2; ++e) {
               .AsPoints()
               .Color("rgb 'blue'")
               .Name("\"cos(x)\", '.AsPoints().Color(\"rgb 'blue'\")'"))
+    .ImageSize(image_dim)
 #if 1      
-.OutputFormat(extensions[e]);
+.OutputFormat(formats[e]);
 #else
-    .OutputFormat("svg");
-#endif
-if (FLAGS_regenerate_golden_graphs) bricks::FileSystem::WriteStringToFile(result, (std::string("golden/gnuplot.") + extensions[e]).c_str());
-#ifndef BRICKS_APPLE // TODO(dkorolev): Figure out how to run this test on Apple.
-if (!e) ASSERT_EQ(result, bricks::FileSystem::ReadFileAsString(std::string("golden/gnuplot.") + extensions[e]));
+    .OutputFormat("svg");  // Although the one below is actually a "png".
 #endif
+if (FLAGS_regenerate_golden_graphs) bricks::FileSystem::WriteStringToFile(result, ("golden/science-" + BRICKS_ARCH_UNAME + '.' + extensions[e]).c_str());
```

Comment:
In the `docu_*` file indenting this line would result in it making it into `README.md`, which is undesirable.

I could break it into several unindented lines though, if you think it's worth it.